protos/IncomingPushMessageSignal.proto

```diff
@@ -143,19 +145,19 @@ message SyncMessage {
     optional uint64 timestamp = 2;
   }
 
-  message Settings {
+  message Configuration {
     optional bool readReceipts = 1;
   }
 
-  optional Sent     sent     = 1;
-  optional Contacts contacts = 2;
-  optional Groups   groups   = 3;
-  optional Request  request  = 4;
-  repeated Read     read     = 5;
-  optional Blocked  blocked  = 6;
-  optional Verified verified = 7;
-  optional bytes    padding  = 8;
-  optional Settings settings = 9;
+  optional Sent          sent          = 1;
+  optional Contacts      contacts      = 2;
+  optional Groups        groups        = 3;
+  optional Request       request       = 4;
+  repeated Read          read          = 5;
+  optional Blocked       blocked       = 6;
+  optional Verified      verified      = 7;
+  optional Configuration configuration = 9;
```

Comment:
This will require code changes for sure.

Reply:
Got it: 91612880a5bf2c9ae8a9334877ac24e91102b905